modules.d/01systemd-coredump/module-setup.sh

```diff
@@ -51,6 +51,7 @@ install() {
     # Install the hosts local user configurations if enabled.
     if [[ $hostonly ]]; then
         inst_multiple -H -o \
+            "$systemdutildir"/coredump.conf \
```

Comment:
> Is the following missing ?
> "$systemdutildir"/coredump.conf.d/*.conf" \

Yes, and the other is already added:
https://github.com/dracut-ng/dracut-ng/blob/1e802f15fee3d6402d83e0efefef8bb88c5a33e3/modules.d/01systemd-coredump/module-setup.sh#L37

Reply:
I think the scheme with latest 256 systemd should be just similar. /usr stuff included always, /etc for hostonly.

But

At least in the rhel/centos stream, we set by default #Storage=external, which is, in my opinion, not a good default for initrd. The core-dump is stored in /var, which is effectively deleted after the switch root. Maybe storage=journal is a better default, and we should have our own default conf file?